### ISSUE TITLE
How to train lora weights about camera movement controls?

### ISSUE THREAD

**Issue by YisuiTT** [guoyww/AnimateDiff#160]
This work is great, but when to publish the code on training lora weights.

**Comment by thiswinex** [comment]
I guess it's trained under animatediff. We can use the same training code with video data about camera pose adjustments. But we have to freeze the motion module and use the same approximate technique as lora.

**Comment by YisuiTT** [comment]
> I guess it's trained under animatediff. We can use the same training code with video data about camera pose adjustments. But we have to freeze the motion module and use the same approximate technique as lora.

I agree with you, but I would like to confirm whether this approach can be adapted to more personalized camera movements.
